CMakeLists.txt

```diff
@@ -267,7 +267,7 @@ if (OPM_ENABLE_PYTHON)
     endif()
     set(PYTHON_INCLUDE_DIRS ${PYTHON_INCLUDE_DIR})
   endif()
-  find_package(pybind11 2.2 CONFIG)
+  find_package(pybind11 2.11 CONFIG)
```

Comment:
Increased the version from 2.2 to 2.11 here

Reply:
that's a very bad idea. this is a minimum version, and a lot of us use system packages. i suggest keeping this at 2.2 and bump if we ever add something that actually needs the newer version, or at the very least only bump it to 2.4 which is the version in use on ubuntu 20.04.

Reply:
Agree,  so you install pybind11 as a system package then? I could not find the package on ubuntu, I tried `pip install pybind11` but cmake was not able to find that installation. Anyway I think the solution in https://github.com/OPM/opm-common/pull/3671 is better so I suggest we close this PR.